bokehjs/test/unit/models/tools/inspectors/hover_tool.ts

```diff
@@ -0,0 +1,75 @@
+import {expect} from "assertions"
+import {build_view} from "@bokehjs/core/build_views"
+import {ColumnDataSource} from "@bokehjs/models/sources/column_data_source"
+import {Circle, CircleView} from "@bokehjs/models/glyphs/circle"
+import {GlyphRenderer} from "@bokehjs/models/renderers/glyph_renderer"
+import {Plot} from "@bokehjs/models/plots/plot"
+import {Range1d} from "@bokehjs/models/ranges/range1d"
+
+import {HoverTool, HoverToolView} from "@bokehjs/models/tools/inspectors/hover_tool"
+
+export interface HoverToolTestCase {
+  data: {[key: string]: (number | null)[]}
+  data_source: ColumnDataSource
+  hover_tool_view: HoverToolView
+  glyph_view: CircleView
+}
+
+async function make_testcase(): Promise<HoverToolTestCase> {
+  const plot = new Plot({
+    x_range: new Range1d({start: -1, end: 1}),
+    y_range: new Range1d({start: -1, end: 1}),
+  })
+
+  const plot_view = (await build_view(plot)).build()
+
+  const data = {x: [0, 0.5, 1], y: [0, 0.5, 1], z: [null, null, null]}
+  const data_source = new ColumnDataSource({data})
+
+  const glyph = new Circle({
+    x: {field: "x"},
+    y: {field: "y"},
+  })
+
+  const glyph_renderer = new GlyphRenderer({glyph, data_source})
+  const glyph_renderer_view = await build_view(glyph_renderer, {parent: plot_view})
+
+  const hover_tool = new HoverTool({
+    active: true,
+    renderers: [glyph_renderer as any],
```

Comment:
Don not use `any` type. If there's a type checking issue (there shouldn't be), leave this without a cast for us to figure this out.